src/hotspot/share/utilities/globalDefinitions_visCPP.hpp

```diff
@@ -48,6 +48,11 @@
 #define _USE_MATH_DEFINES
 # include <math.h>
 
+// Only 64-bit Windows is supported
+#ifndef _LP64
+#error unsupported platform
+#endif
+
```

Comment:
Does Windows Aarch64 define _LP64?

Reply:
Yes. As Julian says, it's something we set up in our builds:
```
  if test "x$FLAGS_CPU_BITS" = x64; then
    $1_DEFINES_CPU_JDK="${$1_DEFINES_CPU_JDK} -D_LP64=1"
    $1_DEFINES_CPU_JVM="${$1_DEFINES_CPU_JVM} -D_LP64=1"
  fi
```

Reply:
Ugghh! I was thrown by the `x` in test expressions and thought `x64` meant, well, x64 :)

Reply:
Yeah, that idiom is indeed difficult to read. :-(